packages/Popover/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { memo } from 'react'
+import { bool, func, node, object } from 'prop-types'
+import {
+  PopoverArrow,
+  PopoverDisclosure,
+  usePopoverState as reakitUsePopoverState
+} from 'reakit/Popover'
+import { Box } from '@welcome-ui/box'
+import { Button } from '@welcome-ui/button'
+import { CrossIcon } from '@welcome-ui/icons.cross'
+
+import * as S from './styles'
+
+export function usePopoverState(options) {
+  return reakitUsePopoverState(options)
+}
```

Comment:
To simplify it a bit:
```suggestion
export function usePopoverState = reakitUsePopoverState
```

Reply:
👍